### PR TITLE
Feature/cx 5307 customisations modal background, border, info header pills

### PR DESCRIPTION
**Pull request by kopijunkie**
# Task
We want to offer customers supported customisation options for the Web SDK so that they can customise 
- the background colour of the SDK modal container 
- the SDK modal, information container border style, colour
- information header pills
Default Web SDK Cross Device Flow - Mobile Device Connected screenDefault Web SDK Camera Permission Recovery screen

Customised Web SDK Cross Device Flow - Mobile Device Connected screenCustomised Web SDK Camera Permission Recovery screen

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [x] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1346-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1346-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1346-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1346-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1346-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1346-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1346-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1346-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1346-pr-onfido-sdk-ui-onfido.surge.sh

**Review by ethan605 — APPROVED** [review]
LTGM 🙏

File: src/components/CameraPermissions/Recover/style.scss [hunk 1]
```diff
@@ -22,12 +19,9 @@
   margin: 0 0 8 * $unit-small;
 }
 
+/* Camera Permission "Recovery" header pill, has subtle differences to the Theme header pill */
```
Comment (stefaniacardenas) [inline src/components/CameraPermissions/Recover/style.scss:22]: I don't remember what the design looks like. I wonder if makes sense to make these two pills consistent?

**Review by stefaniacardenas — APPROVED** [review]
LGTM! Only a minor comment that's more of a question for designers

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1346-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1346-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1346-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/CameraPermissions/Recover/style.scss (hunk 1 above)
Comment (kopijunkie) [reply src/components/CameraPermissions/Recover/style.scss:22]: Have addressed this and have made adjustments to make the pill on both screens use the same style.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1346-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1346-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1346-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**